linkinator.config.json

```diff
@@ -3,7 +3,8 @@
   "skip": [
     "https://codecov.io/gh/googleapis/",
     "www.googleapis.com",
-    "img.shields.io"
+    "img.shields.io",
+    "http://169.254.169.254/latest/api/token%22"
```

Comment:
```suggestion
    "img.shields.io"
```

This file is generated, so removing this from ignore -- I don't think we should need it unless we link to this URL somewhere in the ref docs.